packages/core/docs/migrate/projects-2.2.0.md

```diff
@@ -42,4 +42,4 @@ We have changes a bit the naming and signatures of composable functions. Below i
 |      useWishlist      |   removeFromWishlist     |       (product: WISHLIST_ITEM, customQuery?: CustomQuery)       |       ({ product: WISHLIST_ITEM, customQuery?: CustomQuery })        |
 |      useWishlist      |   load     |       (customQuery?: CustomQuery)       |       ({ customQuery?: CustomQuery } = {})        |
 |      useWishlist      |   clearWishlist     |       ()       |       ()        |
-|      useWishlist      |   isOnWishlist     |       (product: PRODUCT)       |       ({ product: PRODUCT })        |
\ No newline at end of file
+|      useWishlist      |   isInWishlist     |       (product: PRODUCT)       |       ({ product: PRODUCT })        |
```

Comment:
This file shouldn't be modified, as it's a migration guide for previous version.

Reply:
Thanks @filipsobol. I removed this change in the latest commit